scripts/deploy.py

```diff
@@ -31,7 +31,12 @@ def get_address(msg: str, default: str = None) -> str:
 
 def main():
     print(f"You are using the '{network.show_active()}' network")
-    dev = accounts.load(click.prompt("Account", type=click.Choice(accounts.load())))
+    actIndex = int(
```

Comment:
what's the issue you are seeing with the prev version?
do you have accounts inside brownie?

Reply:
I do, but it seems `accounts.load()` doesn't do anything whereas just calling `accounts` directly does. Here's a screenshot, lmk if I'm missing something. If not, I wanted to add a little more to this so that raw addresses can be input as well.
![Screenshot from 2022-06-19 23-20-18](https://user-images.githubusercontent.com/100969824/174524700-c14e05b5-a0fb-494b-900e-22fae8b23a2e.png)
.

Reply:
How do you run the scripts?

Reply:
I was using the deploy script with the `brownie run deploy` command.

Reply:
the screenshot is in the console of course, but I get the same result when running the script